protocols/relay/src/behaviour/handler.rs

```diff
@@ -896,33 +779,49 @@ impl ConnectionHandler for Handler {
         >,
     ) {
         match event {
-            ConnectionEvent::FullyNegotiatedInbound(fully_negotiated_inbound) => {
-                self.on_fully_negotiated_inbound(fully_negotiated_inbound)
-            }
-            ConnectionEvent::FullyNegotiatedOutbound(fully_negotiated_outbound) => {
-                self.on_fully_negotiated_outbound(fully_negotiated_outbound)
-            }
-            ConnectionEvent::ListenUpgradeError(listen_upgrade_error) => {
-                self.on_listen_upgrade_error(listen_upgrade_error)
+            ConnectionEvent::FullyNegotiatedInbound(FullyNegotiatedInbound {
+                protocol: stream,
+                ..
+            }) => {
+                self.inbound_requests_futs.push(
+                    inbound_hop::process_inbound_request(
```

Comment:
I'd prefer if we establish a convention of what we call the "stream handler" functions within the `inbound_hop`, `outbound_stop` etc modules. Maybe something like `new_stream` or `handle`?

Reply:
`handle_inbound_request`?